README.md

```diff
@@ -150,6 +150,11 @@ actively developed primarily for iOS. While we can catch basic unit test issues
 may be some changes where the SDK no longer works as expected on macOS or tvOS. If you encounter
 this, please [file an issue](https://github.com/firebase/firebase-ios-sdk/issues).
 
+## Carthage
+
+An experimental Carthage distribution is now available. See
+[Carthage](Carthage.md).
```

Comment:
Maybe worth adding a note (here or somewhere else) that these distributions are still static binaries, and people should be wary of building wrapper frameworks with Carthage on top of them.

Reply:
I added a Static Frameworks section to communicate this ...